doc/GETSTARTED.md

```diff
@@ -0,0 +1,77 @@
+# Get Started
+
+- Skill requirements
+
+  - Operating system
+
+    We will need a unix-based operating system to run Elektra (Linux, BSD, macOS) but it's also possible to use Windows which is supported but not tested much. If you are new to the world of unix we recommend you to use one of the common linux distributions like [Ubuntu](https://ubuntu.com/download/desktop#download), [Linux Mint](https://www.linuxmint.com/download.php), [openSuse](https://software.opensuse.org/distributions/leap) or [Fedora](https://getfedora.org/en/).
```

Comment:
If someone really wants to contribute to Elektra and does not have a unix based system, he will have severe problems learning unix based systems before actually contributing to Elektra. If someone already has experience with unix (which I assume the vast majority of Dev's contributing to C related projects have), these sentences are irrelevant. Therefore I would simply omit those sentences